python/ray/experimental/ui.py

```diff
@@ -312,42 +377,57 @@ def handle_submit(sender):
         diff = largest - smallest
 
         if time_opt.value == total_time_value:
-            tasks = ray.global_state.task_profiles(
-                start=smallest + diff * low,
-                end=smallest + diff * high)
+            tasks = ray.global_state.task_profiles(start=smallest + diff * low,
+                                                   end=smallest + diff * high)
         elif time_opt.value == total_tasks_value:
             if range_slider.value[0] == 0:
-                tasks = ray.global_state.task_profiles(
-                    num_tasks=int(num_tasks * high),
-                    fwd=True)
+                tasks = ray.global_state.task_profiles(num_tasks=int(
+                                                       num_tasks * high),
+                                                       fwd=True)
             else:
-                tasks = ray.global_state.task_profiles(
-                    num_tasks=int(num_tasks * (high - low)),
-                    fwd=False)
+                tasks = ray.global_state.task_profiles(num_tasks=int(
+                                                       num_tasks *
+                                                       (high - low)),
+                                                       fwd=False)
         else:
-            raise ValueError(
-                "Unexpected time value '{}'".format(time_opt.value))
-
+            raise ValueError("Unexpected time value '{}'".format(
+                                                            time_opt.value))
+        # Write trace to a JSON file
         print("{} tasks to trace".format(len(tasks)))
-        print("Dumping task profiling data to " + tmp)
-        ray.global_state.dump_catapult_trace(tmp,
+        print("Dumping task profiling data to " + json_tmp)
+        ray.global_state.dump_catapult_trace(json_tmp,
                                              tasks,
-                                             breakdowns=breakdown)
-        print("Converting chrome trace to " + tmp2)
-        trace2html = find_trace2html()
-        # TODO(rkn): The trace2html script currently requires Python 2.
-        # Remove this dependency.
```

Comment:
can we move this comment over to the place where we are calling `python2` on `vulcanize_bin`?